tox.ini.jinja

```diff
@@ -10,18 +10,21 @@ commands = pytest
 
 [testenv:docs]
 description = invoke sphinx-build to build the HTML docs
-deps =
-  {posargs:}
-  -r requirements/docs.txt
+deps = -r requirements/docs.txt
 allowlist_externals=find
-setenv =
-  {[testenv]setenv}
-  # Search directory for confuse for custom config enabling plopp plotting
-  SCIPPDIR={toxinidir}{/}docs{/}buildconfig
 commands = python -m sphinx -j2 -v -b html -d {toxworkdir}/docs_doctrees docs html
            python -m sphinx -j2 -v -b doctest -d {toxworkdir}/docs_doctrees docs html
            find html -type f -name "*.ipynb" -not -path "html/_sources/*" -delete
 
+[testenv:releasedocs]
+description = invoke sphinx-build to build the HTML docs from a released version
+skip_install = true
+deps =
+  {{projectname}}=={posargs}
+  {[testenv:docs]deps}
+allowlist_externals={[testenv:docs]allowlist_externals}
+commands = {[testenv:docs]commands}
+
```

Comment:
This was needed so I could avoid having to pass the projectname from the workflow.

Reply:
I think at some point we added this "dependency" in PyPI to ensure we do not upload new docs unless the package was actually published. We can consider changing this, please open a new issue if you think we should.

Reply:
See https://github.com/scipp/copier_template/issues/38